LabBuilder/LabBuilder.psm1

```diff
@@ -1484,16 +1598,19 @@ function Initialize-LabVMTemplates {
         {
             # The template VHD isn't in the VHD Parent folder - so copy it there after optimizing it
             if (-not (Test-Path $VMTemplate.sourcevhd))
-            {
+            {  
+				$VMTemplateDisks = Get-LabVMTemplateDisks -Config $Configuration
+				Initialize-TemplateVHD -Config $Configuration  -VMTemplateDisks $VMTemplateDisks
+
                 # The source VHD does not exist - so try and create it from the ISO
                 # This feature is not yet supported so will throw an error
-                $ExceptionParameters = @{
-                    errorId = 'TemplateSourceVHDNotFoundError'
-                    errorCategory = 'InvalidArgument'
-                    errorMessage = $($LocalizedData.TemplateSourceVHDNotFoundError `
-                        -f $VMTemplate.name,$VMTemplate.sourcevhd)
-                }
-                New-LabException @ExceptionParameters
+                #$ExceptionParameters = @{
```

Comment:
Feel free to remove this code completely - because of source control we can always look back and see what was there before. :smile: